tox.ini

```diff
@@ -0,0 +1,32 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py
+
+[testenv]
+# Pure software tests
+# Invoke as `tox`, `tox -e py27` etc.
+deps =
+    -r{toxinidir}/test_requirements.txt
+commands_pre =
+    python --version
+commands =
+    python -m unittest discover {posargs}
+
+[testenv:integration-test]
+# Tests with real YubiKeys
+# Invoke as `tox -e integration-test`
+basepython = python
+deps =
+    {[testenv]deps}
+    pytest
+commands_pre =
+    {[testenv]commands_pre}
+commands =
+    python setup.py test
```

Comment:
Should this syntax change as well?

Reply:
Yes, eventually, but right now this remains the only way to run the integration tests.